runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -1586,6 +1592,11 @@ J9::ValuePropagation::isArrayCompTypeValueType(TR::VPConstraint *arrayConstraint
       return TR_maybe;
```

Comment:
The comment about **Future refinements** above can be removed, as the future is now.  :-)

Reply:
Similarly, the comment about [**Future refinements** in `isValue`](https://github.com/eclipse-openj9/openj9/blob/974bdd6c51cb71fab4ea4ab50b5fab019141577d/runtime/compiler/optimizer/J9ValuePropagation.cpp#L636-L643) can be removed.

Reply:
Future refinements comment is removed in f926136